source/proof_address/lib/proof_address.rb

```diff
@@ -37,6 +37,7 @@ def proof
 
       result = proofer_result.to_h
       result[:context] = { stages: [address: LexisNexis::PhoneFinder::Proofer.vendor_name] }
+      result[:transaction_id] = proofer_result.transaction_id
```

Comment:
does the phone finder already supply this?

Reply:
> does the phone finder already supply this?

Yeah, assuming I'm following the logic correctly, it's assigned in the base class for both Instant Verify and Phone Finder here:

https://github.com/18F/identity-lexisnexis-api-client-gem/blob/415a0e9/lib/lexisnexis/proofer.rb#L5